bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java

```diff
@@ -319,12 +327,12 @@ private String getFormattedParameters(Collection<String> stateNames) {
             sb.append(',');
             sb.append("{\"name\": \"").append(name).append("\"}");
         }
-        logger.debug("Formatted parameters: {}", sb.toString());
+        logger.trace("Formatted parameters: {}", sb.toString());
```

Comment:
Don't call `.toString()` on parameter objects when logging. This will automatically be done by the logger (but only if the configured log level is high enough). This way the relatively expensive `toString` operations are only executed when strictly necessary.

```suggestion
        logger.trace("Formatted parameters: {}", sb);
```

Reply:
done